app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Forgot your password?</h2>
```

Comment:
**You did great job, your confirmabe method is working perfect.** 

- [ ] However, I'm facing an issue with recoverable with your devise. If I enter my email I'm facing with this error. Kindly watch the video. Thank you!


https://user-images.githubusercontent.com/69011963/216399038-1cbf68ad-33de-4c13-813b-9e6dc8a2fe57.mov

Reply:
Thanks for the feedback. This bug has been fixed in my next commit